HWIMO-BUILD

```diff
@@ -9,6 +9,12 @@
 set -e
 set -x
 
+# $1 should be a file which contains the release version.
+# For example:  PKG_VERSION=on-http~1.2.3-rc-abc...
+if [ -f "$1" ];then
```

Comment:
Because we need to reuse the file for other purpose.
Also, I will put more parameters into a file and pass them via the file.
For example: 
DEBEMAIL="hwimo robots <hwimo@hwimo.lab.emc.com>"
DEBFULLNAME="The HWIMO Robots"